### ISSUE TITLE
Error from docs example

### ISSUE THREAD

**Issue by cdeil** [astropy/saba#11]
I was trying out the example from
http://saba.readthedocs.io/en/latest/index.html#uncertainty-estimation
put as a `test.py` file here: https://gist.github.com/cdeil/69eaf0c9979d47570c2a2b25dc3afccb 

I'm getting this error:

```
$ python test.py 
Traceback (most recent call last):
  File "test.py", line 23, in <module>
    param_errors = sfit.est_errors(sigma=3)
  File "/Users/deil/software/anaconda/envs/py2/lib/python2.7/site-packages/saba/main.py", line 415, in est_errors
    self.error_info = self._fitter.est_errors(methoddict=methoddict, parlist=parlist)
  File "/Users/deil/software/anaconda/envs/py2/lib/python2.7/site-packages/sherpa/fit.py", line 58, in run
    result = func(fit, *args, **kwargs)
  File "/Users/deil/software/anaconda/envs/py2/lib/python2.7/site-packages/sherpa/fit.py", line 1235, in est_errors
    raise EstErr('rstat>max', str(self.estmethod.max_rstat))
sherpa.utils.err.EstErr: reduced statistic larger than 3
```

Can you reproduce?
Can you fix the docs or code?

**Comment by cdeil** [comment]
For the example here:
http://saba.readthedocs.io/en/latest/examples_2d.html
I'm getting this error:

```
In [6]: sfit = SherpaFitter(statistic=Chi2)
   ...: fitmo = truth.copy()
   ...: fitmo.x_mean = 3650
   ...: fitmo.y_mean = 4250
   ...: fitmo.x_stddev = 100
   ...: fitmo.y_stddev = 100
   ...: fitmo.theta = 10
   ...: fitmo.amplitude = 50
   ...: 
   ...: fitmo = sfit(fitmo, x=x0.flatten(), y=x1.flatten(), z=mexp.flatten()+merr.flatten(),
   ...:              xbinsize=np.ones(x0.size)*dx, ybinsize=np.ones(x1.size)*dx,
   ...:              err=merr.flatten()+np.random.uniform(-0.5,0.5,x0.size))
   ...: 
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-6-04643adb15b3> in <module>()
----> 1 sfit = SherpaFitter(statistic=Chi2)
      2 fitmo = truth.copy()
      3 fitmo.x_mean = 3650
      4 fitmo.y_mean = 4250
      5 fitmo.x_stddev = 100

NameError: name 'Chi2' is not defined
```

**Comment by cdeil** [comment]
For the examples here there's missing model imports:
http://saba.readthedocs.io/en/latest/examples_complex.html#usage-details
http://saba.readthedocs.io/en/latest/examples_mcmc.html#using-sherpa-s-mcmc-sampler

@nocturnalastro - Could you please fix up the docs examples?

One more suggestion: when you have multiple commands with output like in this section:
http://saba.readthedocs.io/en/latest/examples_complex.html#fitting-config
I think it's easier to read (quicker to see which command produces what output) if you just put

```
>>> command1
output1
>>> command2
output2
...
```

instead of

```
>>> command1
>>> command2
output1
output2
```

**Comment by nocturnalastro** [comment]
Sorry about that, 
The Chi2 should be "chi2" and is a relic of an older version probably from my notebooks which served as the basis for the docs. 

I think the "rmax" issue might be similar as the fitting-config section was originally in the other section so I probably made it so fitting-config was required, I thought I had updated that when I removed them.

https://github.com/astropy/saba/blob/master/docs/gen_plots.py  is the script used to generate all the plots so might provide the best reference till I update the docs.
